tests/components/FilterSearch.test.tsx

```diff
@@ -668,6 +668,16 @@ describe('search without section labels', () => {
     userEvent.keyboard('{enter}');
     expect(inputNode).toHaveValue('first name 1');
   });
+  
+  it('when an onDropdownInputChange prop is specified, it gets called each time after the input changes and executeFilterSearch does not', async () => {
+    const mockedOnDropdownInputChange = jest.fn();
+    const executeFilterSearch = jest
+      .spyOn(SearchHeadless.prototype, 'executeFilterSearch');
+    renderFilterSearch({ searchFields: searchFieldsProp, onDropdownInputChange: mockedOnDropdownInputChange});
+    userEvent.type(screen.getByRole('textbox'), 'a');
+    await waitFor(() => expect(mockedOnDropdownInputChange).toHaveBeenCalledTimes(1));
+    () => expect(executeFilterSearch).toHaveBeenCalledTimes(0);
```

Comment:
since this is in the form of a callback, I don't think it'll actually execute the check